Formula/jetty-runner.rb

```diff
@@ -17,6 +17,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jetty-runner --version", 1)
```

Comment:
@neutric I can add one more line in test, runner will check for the context file (it's missing), but it should be good sign that runner is working fine.

assert_match "ERROR: No Contexts defined", shell_output("#{bin}/jetty-runner 2>&1", 1)